src/Umbraco.Infrastructure/PropertyEditors/ValueConverters/JsonValueConverter.cs

```diff
@@ -58,7 +58,7 @@ public override object ConvertSourceToIntermediate(IPublishedElement owner, IPub
                 }
                 catch (Exception ex)
                 {
-                    Current.Logger.Error<JsonValueConverter>(ex, "Could not parse the string '{JsonString}' to a json object", sourceString);
+                    Current.Logger.LogError(ex, "Could not parse the string '{JsonString}' to a json object", sourceString);
```

Comment:
One more - so guess there's a few.  General point then is whether we should be looking to remove use of `Current.Logger` as part this PR.